lib/manageiq/postgres_ha_admin/failover_monitor.rb

```diff
@@ -26,11 +26,13 @@ def monitor
         begin
           connection = pg_connection(handler.read)
           if connection
-            server_store.update_servers(connection)
+            server_store.update_servers(connection, handler.name)
             connection.finish
             next
           end
 
+          log_settings
```

Comment:
How frequently is this getting logged?  This feels like it could be very expensive logging-wise.  Perhaps we only log the first time and if it changes?

Reply:
We log this and the current server store when one of the configuration handlers (rails configuration / region replication) fails to get a connection and we start the failover process with the `do_before_failover` hook.  So, it should only be logged during failover situations.